### ISSUE TITLE
Map image in task table view is not created after coming back online

### ISSUE THREAD

**Issue by bigfish24** [realm-demos/realm-teamwork-MR#24]
In a demo noticed a bug where a client on one device as a manager creates a task to assign to a team member. The team member device is offline while this happens. After the task is created the team members device comes back online and the task appears in the table view. However, the map image for the location is not created and instead the task row has white space looking out of place.

**Comment by dhmspector** [comment]
Right - this is an issue when using the MapKit snapshot functionality it doesn't work offline.

- One fix might be to have a placeholder image for tasks created offline and store images in the realm (right now we create new images at load since they're relatively cheap) and whomever loads it next makes/stores the image. 

- Another idea is to have the back-end always look for missing map snapshot images and create them; this would require MapBox or a similar API and would result in different image styles depending on which side of the system makes the map snapshot unless we found a very similar style between iOS/MapKit and MapBox (or whatever map system we chose).
